mod/researched_metrics/set/right/downvotee_search.rb

```diff
@@ -39,6 +39,20 @@ def search_with_params
     @search_results ||= enrich_result get_search_result
   end
 
+  def enrich_result result
+    result.map do |item_name|
+      # 1) add the main card name on the left
+      # for example if "Apple+metric+*upvotes+votee search" finds "a metric"
+      # we add "Apple" to the left
+      # because we need it to show the metric values of "a metric+apple"
+      # in the view of that item
+      # 2) add "yinyang drag item" on the right
+      # this way we can make sure that the card always exists with a
+      # "yinyang drag item+*right" structure
+      Card.fetch main_name, item_name, "yinyang drag item"
+    end
+  end
```

Comment:
moved here from deleted file.

Reply:
"yingyang drag item"? not sure if that really needed to be saved

Reply:
I had the same thought, but it wasn't obvious to me where to perform the surgery.  It's still possible to get here: https://wikirate.org/Posco+metric+:downvotee_search.

Doesn't do much, but no 500 errors...

Reply:
I assume all the votee stuff will get cleaned up when we move to bookmarking.